src/temporal_terms.jl

```diff
@@ -0,0 +1,56 @@
+# Basic Lag:
+
+# syntax:
+"""
+    lag(term, nsteps)
```

Comment:
```suggestion
    lag(term, nsteps::Integer)

```

BTW, it seems standard to have `nsteps=1` by default (e.g. dplyr, ShiftedArrays...).

Reply:
ok